apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -63,7 +64,13 @@ export default class TutorialExplorer extends React.Component {
 
     const sortBy = props.defaultSortBy;
     const orgName = TutorialsOrgName.all;
-    const filteredTutorials = this.filterTutorialSet(filters, sortBy, orgName);
+    const defaultSearchTerm = '';
```

Comment:
I notice that the default sortBy and initialFilters are coming in through props, would we want to do the same for the searchTerm? Do we want to be able to read the searchTerm from a url param like we do for the initialFilters?

Reply:
Good catch! Maybe? I'm meeting with Amanda to go over the details and will ask about this.

Reply:
We're going to hold off on including a url param for searchTerm at least for now. The idea is to get a sense of usage and whether it meets users' needs as is.